src/v3/src/components/Widget/style.css

```diff
@@ -165,3 +165,12 @@
     filter: unset !important;
   }
 }
+
+span.strong {
```

Comment:
before merging can you put a comment here noting that these are widget styles? the other styles only pertain to the login page